crates/bevy_pbr/src/render/mesh_types.wgsl

```diff
@@ -29,7 +29,11 @@ struct MorphWeights {
 };
 #endif
 
-const MESH_FLAGS_SHADOW_RECEIVER_BIT: u32 = 1u;
-const MESH_FLAGS_TRANSMITTED_SHADOW_RECEIVER_BIT: u32 = 2u;
+// [2^0, 2^16)
+const MESH_FLAGS_VISIBILITY_RANGE_INDEX_BITS: u32 = 65535u;
+// 2^29
+const MESH_FLAGS_SHADOW_RECEIVER_BIT: u32 = 536870912u;
+// 2^30
+const MESH_FLAGS_TRANSMITTED_SHADOW_RECEIVER_BIT: u32 = 1073741824u;
```

Comment:
Why were these changed? This is a bit unclear to me.

Reply:
I thought it was cleaner if the "int" fields and "flags" fields grew from opposite ends of the bitfield toward one another. The integer fields grow from the bottom up, while the flag fields grow from the top down.